src/test/pybind/test_cephfs.py

```diff
@@ -569,6 +569,73 @@ def test_fchmod(testdir):
     cephfs.close(fd)
     cephfs.unlink(b'/file-fchmod')
 
+def test_chown(testdir):
+    fd = cephfs.open('file1', 'w', 0o755)
+    cephfs.write(fd, b'abcd', 0)
+    cephfs.close(fd)
+
+    uid = 1012
+    gid = 1012
+    cephfs.chown('file1', uid, gid)
+    st = cephfs.stat(b'/file1')
+    assert_equal(st.st_uid, uid)
+    assert_equal(st.st_gid, gid)
+
+def test_chown_change_uid_but_not_gid(testdir):
+    fd = cephfs.open('file1', 'w', 0o755)
+    cephfs.write(fd, b'abcd', 0)
+    cephfs.close(fd)
+
+    uid = 1012
+    cephfs.chown('file1', uid, -1)
+    st = cephfs.stat(b'/file1')
+    assert_equal(st.st_uid, uid)
```

Comment:
Should there not be a check that the gid remains unchanged?

Reply:
Done.